include/uv-darwin.h

```diff
@@ -52,6 +57,7 @@
   void* cf_member[2];                                                         \
   int cf_error;                                                               \
   uv_mutex_t cf_mutex;                                                        \
+  enum uv_fs_event_type_flag type_flag;                                       \
```

Comment:
It looks like I didn't even need to do that. Updated to only continue if len == 0 and the event is for the directory